vast/integration/vast_integration_suite.yaml

```diff
@@ -533,9 +533,16 @@ tests:
     steps:
       - command: import -b suricata
         input: data/suricata/eve.json
-      - command: "export json 'pass'"
-      - command: "export json 'pass
-                               | select timestamp, flow_id, src_ip, dest_ip, src_port'"
+      - command: |
+          export json
+            'pass'
+      - command: |
+          export json
+            ' pass
+            /* a comment here */
```

Comment:
Do you think the comment functionality is mature enough to be documented on vast.io? E.g. in [pipelines/syntax](https://vast.io/docs/understand/language/pipelines#syntax) there is an example pipeline with step-by-step instructions afterwards. Maybe those instructions can become pipeline comments instead.

Reply:
Good idea. I made this change in bb5fd5ebc.